### PR TITLE
add: extension store issues are out-of-scope

### PR DESCRIPTION
**Pull request by hanguokai**
As #240 , extension store issues are out-of-scope.

### CODE REVIEW

**Comment by hanguokai** [comment]
I don't know why the Ipr show ❌ , I have link my W3C and github accounts together.
Screen Shot 2022-07-24 at 12 26 00
Screen Shot 2022-07-24 at 12 26 10

**Review by dotproto — APPROVED** [review]
LGTM. This clarification seems like it may help better set expectations.

**Comment by dotproto** [comment]
Since this is a minor a modification to our charter, I want to quickly highlight the relevant line from the [Amendments to this Charter](https://github.com/w3c/webextensions/blob/main/charter.md#amendments-to-this-charter) section of the charter.

>  The group may make simple corrections to the charter such as deliverable dates by the simpler group decision process rather than this charter amendment process.

To that end, I'd like to get LGTMs from the group's editors and chairs before landing this change.

**Review by xeenon — APPROVED** [review]

**Review by zombie — APPROVED** [review]
Looks good, this was our understanding from the beginning, good to be explicit.

**Comment by dotproto** [comment]
While we don't have explicit approval from @mukul-p, he did not raise any concerns with this when we discussed it the 2022-08-18 meeting. For the sake of this PR I'm interpreting that as tacit approval.

**Pull request merged**
